COLLABORATOR_GUIDE.md

```diff
@@ -68,6 +70,10 @@ wide-reaching impact), and focused changes that affect only documentation
 and/or the test suite, may be landed after a shorter delay if they have
 multiple approvals.
 
+For first time contributors, it is recommanded that you ask the author
```

Comment:
1. recommanded -> recommended ?
2. Avoid "you"?

Reply:
Maybe remove the "it is recommended that you"?:

> For first time contributors, ask the author if they have configured...

Reply:
Ack